src/components/EntityInfo.vue

```diff
@@ -4,40 +4,40 @@
       <v-row no-gutters class="pt-3 pb-4">
         <v-col class="flex-column d-flex justify-space-between" cols="12" md="9">
           <header>
-            <!-- Entity Name / IA Title -->
-            <div v-if="!!businessId" id="entity-legal-name" aria-label="Business Legal Name">
-              <span>{{ getLegalName || 'Unknown Name' }}</span>
-            </div>
-            <div v-if="!!tempRegNumber" id="incorp-app-title" aria-label="Incorporation Application Title">
-              <span>{{ getLegalName || GetCorpNumberedDescription(getLegalType)}}</span>
-            </div>
-
-            <!-- Business Description -->
-            <div v-if="!!businessId">
-              <span id="business-subtitle">{{ businessDescription }}</span>
-
-              <span id="limited-restoration" class="ml-3" v-if="isInLimitedRestoration">
-                <span class="ml-3">Active until {{ getLimitedreRestorationActiveUntil || 'Unknown' }}</span>
-                <v-chip
-                  class="primary mt-n1 ml-3 pointer-events-none font-weight-bold"
-                  small label text-color="white"
-                >
-                  LIMITED RESTORATION
-                </v-chip>
-              </span>
+            <template v-if="!!businessId">
+              <!-- First line -->
+              <div id="entity-legal-name" aria-label="Entity Legal Name">
+                {{ getEntityName || 'Unknown Name' }}
+              </div>
```

Comment:
What do you think of creating two components, `EntityInfoBusinessId` and `EntityInfoTempRegId`?  It feels like we're mixing concerns by combining the two.

Reply:
I'll look into this but it is a bit beyond the scope of this PR.

Reply:
I'm working on this (and old "Tombstone" sub-component) atm. I think you'll like what I've done 😁

Reply:
Please see new commit and resolve this conversation if you are satisfied.